common/primitives/src/schema.rs

```diff
@@ -35,6 +38,28 @@ pub enum PayloadLocation {
 	Paginated,
 }
 
+/// Support for up to 16 user-enabled features on a collection.
+#[bitflags]
+#[repr(u16)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Copy, Clone, RuntimeDebug, PartialEq, Eq, Encode, Decode, MaxEncodedLen, TypeInfo)]
+pub enum Grant {
+	/// Items in this collection are transferable.
+	AppendOnly,
+	/// The metadata of this collection can be modified.
+	SignatureRequired,
+}
+
+impl Default for Grant {
+	fn default() -> Self {
+		Self::AppendOnly
```

Comment:
I think AppendOnly as the default is a bit odd. Ideally the default should be the default of underlying type `u16` which is 0